lib/src/Uncertainty/Distribution/SmoothedUniform.cxx

```diff
@@ -124,12 +124,20 @@ Sample SmoothedUniform::getSample(const UnsignedInteger size) const
   return result;
 }
 
+Sample SmoothedUniform::getSampleByQMC(const UnsignedInteger size) const
+{
+  SampleImplementation result(size, 1);
+  const Point u(SobolSequence(1).generate(size).getImplementation()->getData());
+  const Scalar ba = b_ - a_;
+  for (UnsignedInteger i = 0; i < size; ++i)
+    result(i, 0) = a_ + ba * u[i] + sigma_ * DistFunc::rNormal();
```

Comment:
??? getSampleByQMC *should be* a CDF inversion based on a unique Sobol sequence!

Reply:
oh, right, there is no static keyword